internal/services/pam/pam_test.go

```diff
@@ -489,12 +489,14 @@ func TestSetDefaultBrokerForUser(t *testing.T) {
 		username string
 
 		// These are auxiliary inputs that affect the test setup and help control the mock output.
-		noBroker bool
+		noBroker    bool
+		localBroker bool
```

Comment:
Now that we have another parameter, I think it’s rather a broker (string), with a default to mockBrokerGeneratedID, handle specific "does not exist" and "local".